src/singletons/Settings.hpp

```diff
@@ -268,6 +268,7 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting enableSevenTVGlobalEmotes = {"/emotes/seventv/global", true};
     BoolSetting enableSevenTVChannelEmotes = {"/emotes/seventv/channel", true};
     BoolSetting enableSevenTVEventAPI = {"/emotes/seventv/eventapi", true};
+    BoolSetting sendSevenTVActivity = {"/emotes/seventv/sendActivity", true};
```

Comment:
The negative impact of disabling this by default is that users will *never* see your 7TV badge?
Or does it mean that the badge is just more seldomly updated

Reply:
> users will _never_ see your 7TV badge

Yes, if the EventAPI is used exclusively.

> Or does it mean that the badge is just more seldomly updated

This is only the case with the v2 endpoint (only personal emotes won't be updated there). That's why I'd like to keep it as a backup.

Reply:
note: V2 endpoint sunsets on November 1st, from then on it will return a `410 Gone` with the standard JSON error response, you should check this doesn't pose any issues with the current implementation. Otherwise, it's safe to keep while waiting for users to upgrade their clients.

Reply:
Currently, all errors (non 2xx status, network, ...) are ignored when loading badges, so that should be fine.

Reply:
So am I correct in assuming that if I as a user never send my activity, others who exclusively use eventapi for cosmetics will never see any of my 7TV badges?

Reply:
> note: V2 endpoint sunsets on November 1st, from then on it will return a `410 Gone` with the standard JSON error response, you should check this doesn't pose any issues with the current implementation. Otherwise, it's safe to keep while waiting for users to upgrade their clients.

Could you provide an example of what that payload would look like?

Reply:
> others who exclusively use eventapi for cosmetics will never see any of my 7TV badges

This is correct, yes.

> Could you provide an example of what that payload would look like?

Looks like this, error code may be different, there's no error code for a removed endpoint yet
```json
{
  "status_code": 410,
  "status": "Gone",
  "error": "Unknown Route",
  "error_code": 70498,
  "details": { "message": "The API endpoint requested does not exist" }
}
```